### PR TITLE
feat(kbd_timeout): add privileges for kbd_timeout file

### PR DESCRIPTION
**Pull request by tsivinsky**
This pr adds kbd_timeout file to huawei-wmi group and copies file from /etc/default/huawei-wmi as default value.

### CODE REVIEW

**Comment by tsivinsky** [comment]
sorry, if i did something wrong. it's my first PR. don't judge too harshly :)

**Comment by tsivinsky** [comment]
i don't really know tbh, but i think, that it's available only on `feat/kbdlight` branch on Huawei-WMI repo, so I need to conditionally add it to group

File: generic/huawei-wmi-1.1.2/available [hunk 1]
```diff
@@ -6,6 +6,7 @@ readonly WMI=/sys/devices/platform/huawei-wmi/
 readonly CCT=charge_control_thresholds
 readonly DCT=charge_thresholds
 readonly FLS=fn_lock_state
+readonly KBD=kbdlight_timeout
```
Comment (egormanga) [inline generic/huawei-wmi-1.1.2/available:9]: I'd actually name it `$KLT` as there will be more `kbdlight_*` files.

File: generic/huawei-wmi-1.1.2/privilege [hunk 2]
```diff
@@ -7,5 +7,5 @@
 [ -e "$DEF$cct" ] || cp -- "$WMI$cct" "$DEF"
 chmod g=u -- "$DEF$cct"
 
-chgrp huawei-wmi -- "$WMI$cct" "$WMI$FLS"
-chmod g=u -- "$WMI$cct" "$WMI$FLS"
+chgrp huawei-wmi -- "$WMI$cct" "$WMI$FLS" "$WMI$KBD"
+chmod g=u -- "$WMI$cct" "$WMI$FLS" "$WMI$KBD"
```
Comment (egormanga) [inline generic/huawei-wmi-1.1.2/privilege:11]: You can't just do this, as the file might not exist (e.g. an older version of the module is running, or the feature is not available in hardware). Add `[ -e "$WMI$KBD" ] &&`.

**Review by egormanga — CHANGES_REQUESTED** [review]
LGTM otherwise. I'd also add `power_unlock` since we recently implemented that.

P.S. @n3vu0r — yes, this file might not exist on some devices. Also, `feat/kbdlight` does not seem ready to be merged just yet.

File: generic/huawei-wmi-1.1.2/available (hunk 1 above)
Comment (n3vu0r) [reply generic/huawei-wmi-1.1.2/available]: Makes sense, done.

File: generic/huawei-wmi-1.1.2/privilege (hunk 2 above)
Comment (n3vu0r) [reply generic/huawei-wmi-1.1.2/privilege]: Done, also with `$WMI$PUL`. Thanks.

**Comment by n3vu0r** [comment]
Thanks @egormanga for the review. I've merged latest changes into this PR branch and added your suggestions. I didn't test anything and I don't know how these new features work. Can you tell if it makes sense? Do you have a newer device to test this PR?

**Comment by n3vu0r** [comment]
It basically comes down to whether following is true for all devices.

What the controller remembers and hence no default is required:

  * `fn_lock_state`

What the controller forgets and hence needs to be reinstated from defaults:

  * `charge_control_thresholds`
  * `kbdlight_timeout` (not available on all devices)
  * `power_unlock` (not available on all devices)
  * `smart_charge` (not available on all devices, is this a thing?)

All except for the last point is implemented in this branch but not tested.

**Pull request merged**